app/layout.tsx

```diff
@@ -1,42 +1,47 @@
 import "@/app/global.css";
 import { Paper, Box, ThemeProvider } from "@mui/material";
 import Image from "next/image";
+import React from "react";
 import theme from "@/app/theme";
 
 export const metadata = {
-	title: "Game Trucker",
-	description: "Game Trucker",
+  title: "Game Trucker",
+  description: "Game Trucker",
 };
```

Comment:
![image](https://github.com/user-attachments/assets/ae19a581-af77-41d2-9017-c8c253d10826)

:D 
each page should have its own metadata, that would improve SEO of all individual pages.

Something to read [here](https://www.devagroup.pl/blog/czym-sa-meta-tagi-definicja-i-przyklady-znacznikow-meta-w-seo)